packages/opal-client/opal_client/engine/runner.py

```diff
@@ -252,7 +253,9 @@ def command(self) -> str:
         opts = self._options.get_cli_options_dict()
         opts_string = " ".join([f"{k}={v}" for k, v in opts.items()])
         startup_files = self._options.get_opa_startup_files()
-        return f"opa run --server {opts_string} {startup_files}".strip()
+        opa_path = self._options.opa_executable_path
```

Comment:
Lets add a check that the path actually exists and is a file

Reply:
okay